packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -68,6 +73,28 @@ export interface IDocumentSchema {
 	refSeq: number;
 
 	runtime: Record<string, DocumentSchemaValueType>;
+
+	// Other document related properties that are not related to the schema.
+	info: IDocumentSchemaInfo;
+}
+
+/**
+ * Top-level properties of the document that do not relate to the schema itself.
+ * @internal
+ */
+export interface IDocumentSchemaInfo {
+	/**
+	 * minVersionForCollab that was used when this document was created.
+	 * We use this to issue telemetry warning events if a client tries to open a document
+	 * with a runtime version lower than this.
+	 *
+	 * See {@link @fluidframework/container-runtime#LoadContainerRuntimeParams} for additional details on `minVersionForCollab`.
+	 *
+	 * @remarks
+	 * We use `SemanticVersion` instead of `MinimumVersionForCollab` since we may open future documents that with a
+	 * minVersionForCollab version that `MinimumVersionForCollab` does not support.
+	 */
```

Comment:
```suggestion
	/**
	 * The minimum version of the FF runtime that should be used to load this document.
	 * 
	 * We use this to issue telemetry warning events if a client tries to open a document
	 * with a runtime version lower than this.
	 *
	 * See {@link @fluidframework/container-runtime#LoadContainerRuntimeParams} for additional details on `minVersionForCollab`.
	 *
	 * @remarks
	 * We use `SemanticVersion` instead of `MinimumVersionForCollab` since we may open future documents that with a
	 * minVersionForCollab version that `MinimumVersionForCollab` does not support.
	 */
```